devops/scripts/install_drivers.sh

```diff
@@ -8,6 +8,7 @@ if [ -f "$1" ]; then
     CONFIG_FILE=$1
     CR_TAG=$(jq -r '.linux.compute_runtime.github_tag' $CONFIG_FILE)
     IGC_TAG=$(jq -r '.linux.igc.github_tag' $CONFIG_FILE)
+    IGC_DEV_HASH=$(jq -r '.linux.igc_dev.github_hash' $CONFIG_FILE)
```

Comment:
Please drop the "if" on line 6. I don't think we use it. Alternatively, we'd need support in the "else" branch, which would be stupid if we don't really use it :)